src/components/api.jsx

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
```

Comment:
Great that you're using axios and separating out your api requests ✅ 
`getArticles` though doesn't need the `article_id` argument